engine/sync.go

```diff
@@ -544,6 +543,8 @@ func (sc *syncClient) run() error {
 		default:
 			sc.runOnce()
 			select {
+			// TODO: If we receive on quit inside runOnce, we have to wait the
+			// 5s here before enable will work again.
```

Comment:
Maybe document how 5 seconds was chosen?

Reply:
I can't find any clear reason for it back when it was first written, so it may be lost to time. 
To match syncHeartbeatInterval? (But then why not use the const?)